relayer/relays/parachain/beefy-listener.go

```diff
@@ -561,6 +563,17 @@ func (li *BeefyListener) scanForCommitments(
 			"blockNumber": currentBlockNumber,
 		}).Debug("Checking header")
 
+		decodedAccount, err := hex.DecodeString(li.config.Account)
+		if err != nil {
+			return nil, fmt.Errorf("could not decode channel account id: %v", err)
```

Comment:
By convention, we don't include phrases such as "could not", "failed to", or "error while" in error messages. As such phrases end up being repeated redundantly in the final top-level wrapped error message".

Should generally also use the "%w" flag for formatting errors. See https://golangbyexample.com/wrapping-and-unwrapping-error-golang/

```suggestion
			return nil, fmt.Errorf("decode account id: %w", err)
```